vent/common/prefs.py

```diff
@@ -52,6 +54,7 @@
 Declare all available parameters and set default values. If no default, set as None. 
 
 * ``PREFS_FN`` - absolute path to the prefs file
+* ``TIME_FIRST_START`` - time when the program has been started for the first time
```

Comment:
@sneakers-the-rat cool with that? It's a single number that is stored the first time the controller is started. You would have to add a technical alarm current_date - this number exceeds a threshold.